roles/monitoring_server/templates/prometheus_rules.yml.j2

```diff
@@ -3,7 +3,7 @@ groups:
     rules:
```

Comment:
Only the changes in this file are relevant, the others is because the linting workflow was failing. I updated the hooks and hoped that would fix it.